File: Day4/Solution.py
class Person:
    def __init__(self,initialAge):
        # Add some more code to run some checks on initialAge
        if(initialAge > 0):
            self.age = initialAge
        else:
            print("Age is not valid, setting age to 0.")
            self.age = 0

    def amIOld(self):
        # Do some computations in here and print out the correct statement to the console
        if self.age >= 18:
            print("You are old.")
        elif self.age >= 13:
            print("You are a teenager.")
        else: # age < 13
            print("You are young.")

    def yearPasses(self):
        # Increment the age of the person in here
        self.age += 1

File: Day4/Solution.swift
class Person {
    var age: Int = 0
    
    init(initialAge: Int) {
        if initialAge > 0 {
            self.age = initialAge
        } else {
            print("Age is not valid, setting age to 0.")
            self.age = 0
        }
    }
    
    func yearPasses() {
        self.age += 1
    }
    
    func amIOld() {
        if self.age < 13 {
            print("You are young.")
        } else if self.age < 18 {
            print("You are a teenager.")
        } else {
            print("You are old.")
        }
    }
    
}
